src/App.tsx

```diff
@@ -126,6 +176,14 @@ export default function App({ fileId, isEmbedded, fileName }: WhiteboardAppProps
 		)
 	}
 
+	const renderSmartPicker = () => {
+		return (
+			<button className="dropdown-menu-button App-toolbar__extra-tools-trigger" aria-label="Library" aria-keyshortcuts="0" onClick={pickFile} title='Smart Picker'>
+				<Icon path={mdiShape} size={1} />
```

Comment:
I think I missed how the icon is defined but maybe we can use https://pictogrammers.com/library/mdi/icon/slash-forward-box/ instead which is also what Talk uses these days

<img width="265" alt="Screenshot 2024-08-16 at 20 41 33" src="https://github.com/user-attachments/assets/d0cc5a2b-e16d-424b-848d-4d3258a63178">